.env.example

```diff
@@ -0,0 +1,3 @@
+# Rename this file to .env and a channel ID to create the scheduled trigger!
+
+RUNNERS_CHANNEL_ID=
```

Comment:
This is left blank since a placeholder will allow the trigger to be successfully created even if that placeholder isn't a known channel!